src/share/streams/shr_dmodel_mod.F90

```diff
@@ -1576,7 +1552,6 @@ subroutine shr_dmodel_mapSet_dest(smatp,ggridS,gsmapS,nxgS,nygS,ggridD,gsmapD,nx
     type(mct_aVect) :: AVl
     type(mct_aVect) :: AVg
 
-    character(len=32) :: lstrategy
```

Comment:
Since it looks like the strategy optional argument is unused, it should be removed as an argument to this routine to avoid confusion.